firebase-database/firebase-database.gradle

```diff
@@ -73,6 +73,7 @@ dependencies {
     implementation project(':firebase-common')
     implementation project(':firebase-database-collection')
 
+    implementation 'androidx.annotation:annotation:1.1.0'
```

Comment:
Does the upgrade require us to add this?

Reply:
Yes, these are needed to satisfy android lint dependencies check. Also these dependencies are correct because database indeed uses these annotations, they just used to come transitively from gms.basement